pkg/harvester-manager/machine-config/harvester.vue

```diff
@@ -429,6 +482,14 @@ export default {
     test() {
       const errors = [];
 
+      if (!this.userDataIsBase64) {
+        this.value.userData = base64Decode(this.value.userData);
+      }
+
+      if (!this.networkDataIsBase64) {
+        this.value.networkData = base64Decode(this.value.networkData);
+      }
+
```

Comment:
In one case, if the `rke2` created by the user is not created through the UI, then it may be saved in plain text, so we should save it as plain text as well.

`issue`: https://github.com/harvester/harvester/issues/3598#event-8675055710